qemu/tests/mmu_basic.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test to disabling radix MMU mode on guest.
+    Steps:
+    1) There are two options, boot up a native(radix)guest or HPT guest.
+    2) Check the MMU mode in the guest.
+    3) Adding disable radix to guest's kernel line directly then reboot guest.
+    4) Check again the MMU mode in the guest.
+    5) Check guest call trace in dmesg log.
+
+
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+
+    def cpu_has_flags(pattern, content):
```

Comment:
What do you think of the name pattern_search?

Reply:
You want to match specific info from cpuinfo by pattern, so the function name should be clearer.
BTW, I see you don't care what is returned, so it might be better to return `True` and `False`.

Reply:
ok, will change to return ​​True and False.
but if you only look at this function, it can match more than the CPU.
Maybe I can change the function to：
```
    def cpu_info_match(pattern):
        match = re.search(pattern, session.cmd_output("cat /proc/cpuinfo"))
        return True if match else False
```
What do you think ?

Reply:
They are all from `cpuinfo`, so it's okay for me.